cdci_data_analysis/flask_app/app.py

```diff
@@ -201,7 +201,9 @@ def common_exception_payload():
 
     if payload['debug_mode'] == "yes":
         payload['config'] = {
-            'dispatcher-config': remove_nested_keys(app.config['conf'].as_dict(),  ['sentry_url', 'logstash_host', 'logstash_port', 'secret_key'])
+            'dispatcher-config': remove_nested_keys(app.config['conf'].as_dict(),
+                                                    ['sentry_url', 'logstash_host', 'logstash_port', 'secret_key',
+                                                     'mail_password',])
```

Comment:
mail_password is SMTP server password, yes? If yes, it's good to be specific - there are many other possible mail-related passwords. Also not all SMTP servers would need password, e.g. those running on localhost (and also as that used currently to send feedback) usually do not.

Reply:
Thanks, will update!